lib/utils.js

```diff
@@ -32,7 +36,7 @@ exports.isWindows = options => {
   if (options && typeof options.windows === 'boolean') {
     return options.windows;
   }
-  return win32 === true || path.sep === '\\';
+  return win32 === true;
```

Comment:
we don't need to check the path sep, since we already know if we're on windows from the above check.  
And path.sep is only ever \ when on windows https://nodejs.org/api/path.html#path_path_sep

Reply:
> And path.sep is only ever \ when on windows

Naively, yes. But this also gave us a super easy way to explicitly set `\\` as the path separator for tests.

Reply:
Sure, but as below, I don't think setting path.sep should be a supported use case, so using that logic  isn't correct with that in mind